### ISSUE TITLE
Hack Request - 1 minute penalties

### ISSUE THREAD

**Issue by Mhopkinsinc** [Mhopkinsinc/NHL94SNESVault#12]
# User story
The game crashes if you set 1 minute penalties. Would like to fix the code to allow 1 minute penalties.

# Possible Solution
- [x] Identify the code that loads the penalty time.
- [x] Compare the difference when its 1 minute vs 2 minutes
- [x] Fix the code so it works with 1 minute.
- [x] Change All Penalties to use the same value from the Config.ASM file

# Actual Solution
- Trace file from MESEN emulator showed different code paths for 1 vs 2 minutes.
- Created ASM Code JUMP to update RAM values and try to mimic the 2 minute penalty behavior.
- Made the 1 minute penalty option configurable in the Config.ASM file
- 0F7B5B new ASM code to try and fix 1 minute penalties.

# Acceptance criteria
- [x] Game Play - Rom doesn't crash when a 1 minute penalty happens.
- [x] Game Play - 1 minute penalty shows on the score board.
- [x] Game Play - 1 minute penalty shows in the in game clock.
- [x] Player Stats - [PM] is correctly displaying 1 and incrementing properly if they have multiple penalties.
- [x] Game Stats - [PM] is correctly displaying 1 the sum of all the players penalty minutes.
- [x] Rom Config - Option is configurable in the Config.asm file when building the ROM.
- [x] Updated SNES NHL94 ROM-RAM Mapping Project with any new values.
- [x] DiztinGUIsh DizRaw file is up to date and has any necessary comments with changes.
- [x] Provided brief description of the actual changes (Solution) performed.

# Sprint Ready Checklist 
- [x] Acceptance criteria defined.
- [x] Identified solution / steps to satisfy acceptance criteria.
- [x] Acceptance criteria is verifiable / testable.

**Comment by Mhopkinsinc** [comment]
```
One_Min_Pen: 
;WDM #$01     ; Used for Debug in emulator
PHA                 ; Push the Accumulator onto the stack before we start
LDA.B $AD       ; Load the value at memory location $0000AD into the Accumulator
CMP.W #$203C ; Compare the Accumulator with the immediate value $20C3 // C3 is 60 in decimal 1 minute pen.
BNE .not1min     ; If the values are not equal, branch to the code labeled notEqual
LDA.W #$0001   ; Load the immediate value $0001 into the Accumulator
STA.W $15DA     ; Store the value in the Accumulator at memory location $15DA                       
PLA                     ; Pull the Accumulator off the stack before RTS
RTS                     ; Return from Subroutine

.not1min:                        
PLA                   ; Pull the Accumulator off the stack
LDA.W $15DA   ; Load the value at memory location $15DA into the Accumulator
RTS ; Return from Subroutine
```

**Comment by Mhopkinsinc** [comment]
graphic glitch, will create a new issue to try and resolve it.
